sklearn/neighbors/classification.py

```diff
@@ -366,8 +366,8 @@ def predict(self, X):
 
         y_pred = np.empty((n_samples, n_outputs), dtype=classes_[0].dtype)
         for k, classes_k in enumerate(classes_):
-            pred_labels = np.array([_y[ind, k] for ind in neigh_ind],
-                                   dtype=object)
+            pred_labels = np.zeros(len(neigh_ind), dtype=object)
```

Comment:
`np.empty`? but `np.zeros` also doesn't allocate memory, so it's fine I suppose?